myRental/test/testRequests.js

```diff
@@ -0,0 +1,168 @@
+let faker = require('faker');
+let crypto = require('crypto');
+let config = require('../config.json');
+
+function generateSearchRequest() {
+    return {
+        'body': {
+            'max': 3,
+            'address': '800-820 E 15th St N',
```

Comment:
Is there a way to use faker to generate these addresses? Does it even need to look like an address for testing purposes?

Reply:
Same as comment above.